tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# tasks file for ansible-role-solr-prometheus-exporter
+
+- name: Solr_prometheus | Copy service template
+  template:
+    src: prometheus_solr_exporter.service.j2
+    dest: /etc/systemd/system/prometheus_solr_exporter.service
```

Comment:
What will happen if there's more than one instance of solr running (which is a pretty common occurrence on our servers)? Would we need to allow for more than one exporter service too?

Reply:
yeah, we will need to. Shall I make the solr directory variable a list and create templates for each of those?

Reply:
and its done.